### PR TITLE
Stronger in!

### PR DESCRIPTION
**Pull request by nikcorg**
Merge in changes by MG.

### CODE REVIEW

File: src/__tests__/__snapshots__/fixtures-test.ts.snap [hunk 1]
```diff
@@ -1340,90 +1092,59 @@ Array [
...
   Object {
     "category": 1,
-    "code": 2454,
-    "length": 4,
-    "messageText": "Variable 'type' is used before being assigned.",
-    "start": 9170,
-    "text": "type",
+    "code": 2693,
+    "length": 7,
+    "messageText": "'Promise' only refers to a type, but is being used as a value here.",
```
Comment (nikcorg) [inline src/__tests__/__snapshots__/fixtures-test.ts.snap]: This looks like the test didn't work properly.

File: src/__tests__/__snapshots__/fixtures-test.ts.snap [hunk 2]
```diff
@@ -1340,90 +1092,59 @@ Array [
...
-    },
-    "start": 9684,
-    "text": "{
-    name: 'size',
-    type: FloatType,
-    resolve: () => Promise.resolve('not a number!'),
-  }",
+    "code": 2693,
+    "length": 7,
+    "messageText": "'Promise' only refers to a type, but is being used as a value here.",
```
Comment (nikcorg) [inline src/__tests__/__snapshots__/fixtures-test.ts.snap]: This looks like the test didn't work properly.

File: src/__tests__/__snapshots__/fixtures-test.ts.snap (hunk 1 above)
Comment (nikcorg) [reply src/__tests__/__snapshots__/fixtures-test.ts.snap]: Fixed.

File: src/__tests__/__snapshots__/fixtures-test.ts.snap (hunk 2 above)
Comment (nikcorg) [reply src/__tests__/__snapshots__/fixtures-test.ts.snap]: Fixed.

**Review by leemhenson — APPROVED** [review]

**Pull request merged**